src/pages/components/CertificateForm.js

```diff
@@ -100,16 +101,19 @@ export default function CertificateForm(props) {
             production_end_time: currentProductionEndTime,
           } = resLocal
           update({
+            [current]: resLocal, // persist as a cache
```

Comment:
this is for later at the moment is just a comment for things that we discussed earlier e.g. CO2, issued - coming back to the form.

Reply:
Can you explain how you are planning to implement this later? If not, then comment it out?

Reply:
This looks good but somehow the left hand persona switch navigation panel is behaving strange .. I can't switch to other personas after finishing the form .. i can tell you separately (off-line) how to replicate this ...

![image](https://github.com/digicatapult/sqnc-hyproof-client/assets/58742260/eb069653-f35d-4562-9e94-6c9105225239)

Reply:
Basically it works until form filled and the PR that I’ve approved redirects to form right away and after filling renders that plain text where it also disables persona switch due to state management. I’ve already created a story to address and mentioned in the stand up https://digicatapult.atlassian.net/jira/software/projects/HYP/boards/190/backlog?selectedIssue=HYP-143
same with current version attempt to switch after form filling and seeing that plain text as per screenshot below

see screenshots, this is after filling a form. This is something still to do and we have not even created a certificate view, so with my new route `/create` this was addressed, now main page is form, where it could have been `/certificvate` create.
<img width="792" alt="Screenshot 2024-02-23 at 1 05 59 pm" src="https://github.com/digicatapult/sqnc-hyproof-client/assets/11794402/d2f19d38-3f42-45ea-bdab-c0c20152de74">
<img width="724" alt="Screenshot 2024-02-23 at 1 05 31 pm" src="https://github.com/digicatapult/sqnc-hyproof-client/assets/11794402/9e9bd253-a697-4997-bf5d-893120a8abc8">